docs/content/modules/tensor_mechanics/systems/Materials/HyperbolicViscoplasticityStressUpdate.md

```diff
@@ -1,10 +1,10 @@
-#RecomputeRadialReturnHyperbolicViscoplasticity
-!description /Materials/RecomputeRadialReturnHyperbolicViscoplasticity
+#HyperbolicViscoplasticityStressUpdate
+!description /Materials/HyperbolicViscoplasticityStressUpdate
 
-!devel /Materials/RecomputeRadialReturnHyperbolicViscoplasticity float=right width=auto margin=20px padding=20px background-color=#F8F8F8
+!devel /Materials/HyperbolicViscoplasticityStressUpdate float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
 ## Description
-{!content/modules/tensor_mechanics/common_documentation/supplementalRecomputeRadialReturn.md!}
+{!content/modules/tensor_mechanics/common_documentation/supplementalRadialReturnStressUpdate.md!}
```

Comment:
This is including the page content? We could just link to the page, but I have no strong opinion on this.